libsolidity/interface/StandardCompiler.cpp

```diff
@@ -146,6 +146,47 @@ Json::Value collectEVMObject(eth::LinkerObject const& _object, string const* _so
 	return output;
 }
 
+bool isTargetRequired(Json::Value const& _targets, string const& _target)
+{
+	for (auto const& target: _targets)
+		if (target == _target)
```

Comment:
@chriseth do you have any code suggestion for doing so? E.g. requiring `evm.assembly` matches both the user supplied `evm.assembly`, `evm.*` and `evm`, without hardcoding these paths.